### ISSUE TITLE
Should _shouldSubmitScore be removed from properties.schema for legacy AT users?

### ISSUE THREAD

**Issue by deltanetdan** [adaptlearning/adapt-contrib-spoor#225]
[This commit ](https://github.com/adaptlearning/adapt-contrib-spoor/commit/0c551248b6d2e0d9e46907a864dcf5f76c4d5e47) removed _shouldSubmitScore from config.schema.json. 

Should _shouldSubmitScore be removed from [properties.schema ](https://github.com/adaptlearning/adapt-contrib-spoor/blob/master/properties.schema#L33) as well?

**Comment by oliverfoster** [comment]
Yes. Apparently my brain isn't working properly. I should have removed it from properties.schema initially.
